optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/ReportHelper.java

```diff
@@ -47,7 +47,16 @@ public static String escapeHtmlId(String rawHtmlId) {
     }
 
     public static String capitalize(String s) {
-        return StringUtils.capitalize(s);
+        if (Objects.isNull(s)) {
+            return null;
+        }
+        if (s.isEmpty()) {
+            return s;
+        }
+        StringBuilder sb = new StringBuilder();
+        sb.append(Character.toTitleCase(s.charAt(0)));
+        sb.append(s.substring(1));
+        return sb.toString();
```

Comment:
Although I want to get rid of the commons-lang3 dependency if JDK 8 can support all those methods, I don't want to get rid of it by writing the methods ourselves - I think it's better to keep the commons-lang3 dependency then.